### PR TITLE
CLD-2502: chore(download) migrate downloads off of bintray

### PR DESCRIPTION
**Pull request by hutchic**
https://deploy-preview-2668--kongdocs.netlify.app/install/centos/
https://deploy-preview-2668--kongdocs.netlify.app/install/ubuntu/

### Review

@reviewer

### CODE REVIEW

File: app/install/aws-linux.md [hunk 1]
```diff
@@ -16,22 +16,17 @@ This guide walks through downloading, installing, and starting **Kong Gateway**
 
 Start by downloading the following package specifically built for the Amazon Linux AMI:
 
-- [Download]({{ site.links.download }}/kong-rpm/download_file?file_path=amazonlinux/amazonlinux2/kong-{{site.data.kong_latest.version}}.aws.amd64.rpm)
+- [CentOS 8]({{ site.links.download }}/kong-rpm/aws/2/kong-{{site.data.kong_latest.version}}.aws.amd64.rpm)
 
 **Enterprise trial users** should download their package from their welcome email and save their license to `/etc/kong/license.json` after step 1.
```
Comment (lena-larionova) [inline app/install/aws-linux.md:21]: This doesn't exist anymore, so we can toss this line.
```suggestion
```

File: app/install/aws-linux.md [hunk 2]
```diff
@@ -16,22 +16,17 @@ This guide walks through downloading, installing, and starting **Kong Gateway**
...
 ## YUM Repositories
 
-You can also install Kong via YUM; follow the instructions on the "Set Me Up"
-section on the page below.
-
-- [RPM Repository](https://bintray.com/kong/kong-rpm)
-
-**NOTE**: ensure that the `baseurl` field of the generated `.repo` file contains
-amazonlinux/amazonlinux; for instance:
+You can also install Kong via YUM
```
Comment (lena-larionova) [inline app/install/aws-linux.md:25]:
```suggestion
You can also install Kong via YUM:
```

File: app/install/aws-linux.md [hunk 3]
```diff
@@ -16,22 +16,17 @@ This guide walks through downloading, installing, and starting **Kong Gateway**
...
 
 ```
-baseurl=https://kong.bintray.com/kong-rpm/amazonlinux/amazonlinux
+curl {{ site.links.download }}/kong-rpm/aws/2/config.repo | tee /etc/yum.repos.d/kong.repo
+yum install -y kong
```
Comment (lena-larionova) [inline app/install/aws-linux.md:29]: This is two commands, yeah? Should split this up, that's not super clear from the codeblock. Something like

1. Pull down the yum repo:
codeblock
2. Install the Kong repo (is that correct?):
codeblock

File: app/install/aws-linux.md [hunk 4]
```diff
@@ -50,9 +45,9 @@ baseurl=https://kong.bintray.com/kong-rpm/amazonlinux/amazonlinux
...
     $ sudo yum update -y
     $ sudo yum install -y wget
-    $ wget https://bintray.com/kong/kong-rpm/rpm -O bintray-kong-kong-rpm.repo
-    $ sed -i -e 's/baseurl.*/&\/amazonlinux\/amazonlinux'/ bintray-kong-kong-rpm.repo
-    $ sudo mv bintray-kong-kong-rpm.repo /etc/yum.repos.d/
+    $ wget {{ site.links.download }}/kong-rpm/rpm -O kong.repo
+    $ sed -i -e 's/baseurl.*/&\/amazonlinux\/amazonlinux'/ kong.repo
+    $ sudo mv kong.repo /etc/yum.repos.d/
     $ sudo yum update -y
     $ sudo yum install -y kong
```
Comment (lena-larionova) [inline app/install/aws-linux.md:52]: What's the difference between this install section and the previous install section in the same topic?

File: app/install/centos.md [hunk 5]
```diff
@@ -7,60 +7,43 @@ breadcrumbs:
...
 
-1. **Install Kong**
-
-    If you are downloading the [package](#packages), execute:
-
+1. **Installation**
+
+    Kong can be installed by downloading an installation package or from our yum repository
+    
+    **Packages**
```
Comment (lena-larionova) [inline app/install/centos.md:16]: Maybe we can put these sections into tabs? Tabs are nice for either/or situations like this one.

Here's what it looks like after a quick test:
Screen Shot 2021-04-13 at 5 08 36 PM

Navtab code:
```
{% navtabs %}
{% navtab Download packages %}

content

{% endnavtab %}
{% navtab YUM repo %}

content

{% endnavtab %}
{% endnavtabs %}
```

File: app/install/centos.md [hunk 6]
```diff
@@ -7,60 +7,43 @@ breadcrumbs:
...
-
 ## Installation
 
-1. **Install Kong**
-
-    If you are downloading the [package](#packages), execute:
-
+1. **Installation**
+
+    Kong can be installed by downloading an installation package or from our yum repository
```
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/install/centos.md:14]: After looking at the rest of the topic, looks like everything else is also just one step each time, so you can remove them from the rest of the content here too.

File: app/install/centos.md [hunk 7]
```diff
@@ -111,17 +94,21 @@ baseurl=https://kong.bintray.com/kong-rpm/centos/7
...
     $ kong start [-c /path/to/kong.conf]
     ```
 
-4. **Use Kong**
+## Use Kong
+
+1. **Use Kong**
 
-    Kong is running:
+    Kong is running
```
Comment (lena-larionova) [inline app/install/centos.md:111]:
```suggestion
    Verify that Kong is running:
```

**Review by lena-larionova — COMMENTED** [review]
I like the combination of the download and install sections, that's already much clearer - you can ignore the comments I left on the AWS Linux topic about that.

Added some suggestions for simplifying down even further.

**Review by lena-larionova — COMMENTED** [review]
(Not commenting on text/content, since it looks like you're not ready for that yet - no point)

I'm liking the navtabs, looks cleaner. And hooray for reducing copypasta! Looks like everything is displaying as it should with the includes.

I was also testing tabs out for the "with a database/without a database" sections, but looks like document-defined links don't work inside tabs for some reason? Ie, `[Configure][configuration]` just looks like that in the output. What do you think? Worth changing the links and putting them into tabs, or leave as is?

File: app/install/centos.md [hunk 8]
```diff
@@ -7,60 +7,43 @@ breadcrumbs:
...
-    $ sudo yum install -y wget
-    $ wget https://bintray.com/kong/kong-rpm/rpm -O bintray-kong-kong-rpm.repo
-    $ export major_version=`grep -oE '[0-9]+\.[0-9]+' /etc/redhat-release | cut -d "." -f1`
-    $ sed -i -e 's/baseurl.*/&\/centos\/'$major_version''/ bintray-kong-kong-rpm.repo
-    $ sudo mv bintray-kong-kong-rpm.repo /etc/yum.repos.d/
-    $ sudo yum update -y
-    $ sudo yum install -y kong
+    $ curl $(rpm --eval "{{ site.links.download }}/gateway-community/centos/%{centos_ver}/config.repo") | sudo tee /etc/yum.repos.d/kong.repo
+    $ sudo yum info kong
+    $ sudo yum install -y kong-{{site.data.kong_latest.version}}
```
[the remark this answers is not in the document]
Comment (hutchic) [reply app/install/centos.md]: I tried splitting these and couldn't make it aesthetically pleasing

**Comment by tlblessing** [comment]
@hutchic some conflicts need to be resolved (rebase), and not sure if Lena's last review comments have been resolved yet.

File: app/_includes/md/deb.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+$ sudo dpkg -i kong.{{site.data.kong_latest.version}}.amd64.deb
+```
+
+{% endnavtab %}
+{% navtab Repository %}
+
+**APT Repositories**
+
+```bash
+$ sudo apt-add-repository -u 'deb [allow-insecure=yes] {{ site.links.download }}/gateway-community/$(lsb_release -is)/$(lsb_release -sc)/ default all'
```
Comment (hutchic) [inline app/_includes/md/deb.md:38]:
```suggestion
$ sudo apt-add-repository -u "deb [allow-insecure=yes] {{ site.links.download }}/gateway-community/$(lsb_release -is)/$(lsb_release -sc)/ default all"
```

File: app/_includes/md/deb.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+$ sudo dpkg -i kong.{{site.data.kong_latest.version}}.amd64.deb
+```
+
+{% endnavtab %}
+{% navtab Repository %}
+
+**APT Repositories**
+
+```bash
+$ sudo apt-add-repository -u "deb [allow-insecure=yes] {{ site.links.download }}/gateway-community/$(lsb_release -is)/$(lsb_release -sc)/ default all"
```
Comment (hutchic) [inline app/_includes/md/deb.md:38]: this needs to be changed to `echo .... | tee /etc/apt/sources.list`

Comment (HCloward) [reply app/_includes/md/deb.md:38]: @hutchic I'm not sure what needs to be replaced here exactly- are you saying the entire code block starting on line 38 should be: `echo sudo apt-add-repository -u "deb [allow-insecure=yes] {{ site.links.download }}/gateway-community/$(lsb_release -is)/$(lsb_release -sc)/ default all" | tee /etc/apt/sources.list` ?

Comment (hutchic) [reply app/_includes/md/deb.md]: It was more a note to myself. I've updated to reflect production 👍

**Review by HCloward — APPROVED** [review]

**Review by GayleNeumann — APPROVED** [review]
LGTM

**Review by hutchic — COMMENTED** [review]
slack feedback

**Pull request merged**